### ISSUE TITLE
dynamic.MarshalJSONIndent produces spurious leading, trailing commas

### ISSUE THREAD

**Issue by pboyer** [jhump/protoreflect#4]
Input 
```
message msg {
	option (someoption) = "bar";
	uint64 id = 1;
}
```

Output:
 
```
{,
  "id": 12,
}
```

**Comment by jhump** [comment]
Oh weird. I wonder how my tests pass with that! The test basically does a translation party, where it has dynamic message generate JSON and then it uses the core proto `jsonpb` stuff to parse it. And vice versa.

**Comment by jhump** [comment]
Doh! Looks like I wrote code to correctly skip the comma when indenting after `{`. But there was a typo so that it wasn't working properly. Tests didn't catch it because they don't use the indenting versions. (But they should! TODO...)

Fixed in #6

**Comment by pboyer** [comment]
😄
